app/services/early_career_teachers/create.rb

```diff
@@ -30,11 +34,12 @@ def call
 
     attr_reader :full_name, :email, :school_cohort, :mentor_profile_id
 
-    def initialize(full_name:, email:, school_cohort:, mentor_profile_id: nil)
+    def initialize(full_name:, email:, school_cohort:, mentor_profile_id: nil, year_2020: false)
       @full_name = full_name
       @email = email
       @school_cohort = school_cohort
       @mentor_profile_id = mentor_profile_id
+      @year_2020 = year_2020
```

Comment:
Use `attr_reader` like all the other variables here?

Reply:
I'll pick these up tomorrow - for now I just want to stop any more emails being sent. So long as we're happy it will work, that's enough for now

Reply:
Sure, merge when ready :)